controllers/front/createIntent.php

```diff
@@ -96,11 +96,10 @@ private function constructIntentData($amount, $currency, $paymentOption, $paymen
             $shippingAddress = new Address($this->context->cart->id_address_delivery);
             $shippingAddressState = new State($shippingAddress->id_state);
 
-            $metadata = [];
-
-            Hook::exec('actionStripeOfficialMetadataDefinition', $metadata);
-
-            $metadata['id_cart'] = $this->context->cart->id;
+            $metadata = Hook::exec(
+                'actionStripeOfficialMetadataDefinition',
+                ['id_cart' => $this->context->cart->id]
```

Comment:
if actionStripeOfficialMetadataDefinition is not hooked somewhere, $metadata = false

You have a sample here:
https://github.com/shoppingflux/module-prestashop/blob/develop/shoppingfeed.php#L594